torch/backends/mps/__init__.py

```diff
@@ -15,12 +15,16 @@ def is_built() -> bool:
 @_lru_cache()
 def is_available() -> bool:
     r"""Returns a bool indicating if MPS is currently available."""
+    if not hasattr(torch._C, '_is_mps_available'):
+        return False
```

Comment:
Shouldn't be needed, as it works fine right now, see:
```
$ python -c "import torch;import sys;print(sys.platform, torch.__version__, torch.backends.mps.is_available())"
linux 2.0.0a0+git208f330 False

```
```suggestion
```

Reply:
Updated the PR.